packages/venia-ui/lib/components/Navigation/navigation.js

```diff
@@ -51,6 +51,12 @@ const Navigation = props => {
         </Suspense>
     ) : null;
 
+    const switchers = StoreSwitcher ? (
+        <div className={classes.switchers}>
+            <StoreSwitcher mobileView={true} />
+        </div>
+    ) : null;
```

Comment:
Will `StoreSwitcher` ever be falsy? I don't think you need this conditional.

Reply:
It may be when there are only one store view so nothing to switch between, currency switcher may go to `<div class="switchers"/>` too, and they are both independently can be available/unavailable

Reply:
But the export itself will always return the function component. This conditional rendering should probably occur within the switcher component itself.

Reply:
You are right, I need this to have control over `--header-height` custom property, but it's never worked. I have changed to `useRef` to get switchers height.